code/_onclick/hud/lavaland_elite.dm

```diff
@@ -5,7 +5,7 @@
 	..()
 	pull_icon = new /atom/movable/screen/pull()
 	pull_icon.icon = ui_style
-	pull_icon.update_icon()
+	pull_icon.update_appearance(updates = ALL)
```

Comment:
``updates=ALL`` is the default so I think the args should just be empty when you do so. But this is update_icon so it should be ``update_appearance(UPDATE_ICON)``

Reply:
erhm this should still be updating just icon